apps/web/app/CryptoProviders.tsx

```diff
@@ -16,6 +17,7 @@ export type CryptoProvidersProps = {
 
 const config = createConfig({
   chains: [base, baseSepolia, mainnet],
+  connectors: [coinbaseWallet()],
```

Comment:
In OnchainKit we do the following:
```
    coinbaseWallet({
      appName,
      appLogoUrl,
      preference: 'all',
    }),
```

Let's do the same, with appName Base. For the logo, could you make a copy of [this](https://github.com/base/web/blob/d41b9e5938813d50fdc9ab18606c9eca1c04131a/apps/web/src/components/base-org/shared/TopNavigation/assets/logo.svg) in `public/images`?

Reply:
@brendan-defi The name, logo, and preference get overridden by the `onchainKitConfig` defined below. So I think it makes sense to keep this constructor empty since the values won't change anything.

I've added the logo to the onchainkit config: https://github.com/base/web/pull/2280/files#diff-478167902c08c4aa75a24cbbed5c71bd22fb86c77a95725c8d5f9660a3995453R41

![image](https://github.com/user-attachments/assets/98ca945b-763e-46a1-8112-d6cb7c450bf4)